WebKitDev/Functions/Invoke-WebFileRequest.ps1

```diff
@@ -68,9 +94,11 @@ Function Invoke-WebFileRequest {
     }
 
     # Download the file
-    (New-Object System.Net.WebClient).DownloadFile($url, $destinationPath);
+    $tcpClient = New-Object System.Net.WebClient;
+    $tcpClient.Proxy = $proxy;
+    $tcpClient.DownloadFile($url, $destinationPath);
 
-    if ($secure) {
+    if ($secure -and $oldSecurityProtocol) {
```

Comment:
Can security protocol have been "unset" previously and if so should we leave the policy set to whatever we find above or put it back to unset?

Reply:
Good catch!